src/schnapsen/bots/alphabeta.py

```diff
@@ -1,6 +1,6 @@
 from typing import Optional
 
-from schnapsen.game import Bot, Move, PlayerGameState
+from ..game import Bot, Move, PlayerGameState
 
```

Comment:
I thought within the package `from schnapsen.game import Bot, Move, PlayerGameState` will cause an error and a relative import has to be done: `from ..game import Bot, Move, PlayerGameState`. Are both actually okay?

Reply:
Both are okay if the package is installed as instructed in the readme.